app/renderer/actions/Session.js

```diff
@@ -802,29 +802,27 @@ export function setVisibleProviders () {
  * @param {object} caps
  */
 function addCustomCaps (caps) {
-  const {browserName = '', platformName = ''} = caps;
-  const safariCustomCaps = {
-    // Add the includeSafariInWebviews for future HTML detection
-    includeSafariInWebviews: true,
-  };
-  const chromeCustomCaps = {
+  const {platformName = ''} = caps;
+  const androidCustomCaps = {
+    // @TODO: remove when this is defaulted in the newest Appium 1.8.x release
```

Comment:
1.8.x? must be a quite old one

Reply:
That was code I copied over, not sure if we still need it but didn't want to change that in this PR